drivers/ieee802154/ieee802154_cc13xx_cc26xx_subg.c

```diff
@@ -154,33 +137,55 @@ static uint32_t rf_prop_overrides_tx_20[] = {
 	/* Tx: Configure PA ramping, set wait time before turning off */
 	/* (0x1F ticks of 16/24 us = 20.3 us). */
 	HW_REG_OVERRIDE(0x6028, 0x001F),
+	/* Set TXRX pin to 0 in RX/TX and high impedance in idle. */
+	HW_REG_OVERRIDE(0x60A8, 0x0001),
 	(uint32_t)0xFFFFFFFF
 };
 
-/* Radio setup command for CC1312P / CC1352P */
+#else
+#error "unsupported CC13xx SoC"
+#endif /* CONFIG_SOC_CC1352x */
+
+/* Radio setup command for CC13xx */
+#if defined(CONFIG_SOC_CC1352R)
+static volatile rfc_CMD_PROP_RADIO_DIV_SETUP_t ieee802154_cc13xx_subg_radio_div_setup = {
+	.commandNo = CMD_PROP_RADIO_DIV_SETUP,
+#elif defined(CONFIG_SOC_CC1352P)
 static volatile rfc_CMD_PROP_RADIO_DIV_SETUP_PA_t ieee802154_cc13xx_subg_radio_div_setup = {
 	.commandNo = CMD_PROP_RADIO_DIV_SETUP_PA,
+#endif /* CONFIG_SOC_CC1352x */
 	.condition.rule = COND_NEVER,
-	.modulation.modType = 1, /* FSK */
-	.modulation.deviation = 200,
-	.symbolRate.preScale = 15,
-	.symbolRate.rateWord = 131072,
-	.rxBw = 0x59,                   /* 310.8 kHz */
-	.preamConf.nPreamBytes = 7,
-	.formatConf.nSwBits = 24,       /* 24-bit of syncword */
-	.formatConf.bMsbFirst = true,
-	.formatConf.whitenMode = 7,
-	.config.biasMode = true,
-	.formatConf.bMsbFirst = true,
-	.txPower = 0x013f, /* from Smart RF Studio */
-	.centerFreq = 915,
-	.intFreq = 0x0C00,
+	.modulation = {
+		.modType = 1, /* 2-GFSK - non-standard modulation */
+		.deviation = 200, /* +/- 200*250 = 50kHz deviation (modulation index 0.5) */
+	},
+	.symbolRate = {
+		.preScale = 15,
+		.rateWord = 131072, /* 200 kBit, see TRM, section 25.10.5.2, formula 15 */
+	},
+	.rxBw = 0x59, /* 310.8 kHz RX bandwidth, see TRM, section 25.10.5.2, table 25-183 */
+	.preamConf.nPreamBytes = 7, /* phyFskPreambleLength = 7 + 1, also see nSwBits below */
+	.formatConf = {
+		.nSwBits = 24, /* 24-bit (1 byte preamble + 16 bit SFD) */
+		.bMsbFirst = true,
+		.whitenMode = 7, /* Determine whitening and CRC from PHY header */
+	},
+	.config.biasMode = true, /* Rely on an external antenna biasing network. */
+	.txPower = 0x013f, /* 14 dBm, see TRM 25.3.3.2.16 */
+	.centerFreq = 906, /* Set channel page zero, channel 1 by default, see IEEE 802.15.4,
+			    * section 10.1.3.3.
+			    * TODO: Use compliant SUN PHY frequencies from channel page 9.
```

Comment:
Just for clarification, is there already an issue (or perhaps draft PR) in place to cover this?

Reply:
No, not yet, just the generic roadmap in #50336. This is a mandatory pre-condition for the "CH" (aka channel hopping) part in the TSCH acronym.